mixer/pkg/runtime2/testing/data/templates.go

```diff
@@ -18,11 +18,13 @@ import (
 	"context"
```

Comment:
It seems this file is generated template bootstrap code. Unless I am missing something, it seems we are hand editing parts of generated file for the purpose of logging. Do we really need this, especially since we can completely inspect what exactly adapter is receiving or not receiving.

Reply:
No, this is not hand generated code. This is manually written. It is similar to spy adapter, but operates at the template level (as this is the level that runtime2 interacts with). This is meant as a way to mock the template generated code for runtime2 testing purposes.